### ISSUE TITLE
Undocumented key combinations

### ISSUE THREAD

**Issue by ScottRochford** [greenshot/greenshot#106]
I discovered by accident that 'Ctrl + Q' is Drop Shadow in the image editor, which is extremely useful.

However I can't find this or any other keys documented in the help, or the readmes.  I couldn't even find a list in the source code.  Am I missing something obvious?  If not, could it please be added?

The only ones I can find are the capture combinations shown on the system try right-click menu, and the few that are listed beside their respective menu items in the image editor menus, and a smattering of others mentioned in the 'Using the image editor' section of the help page.

**Comment by ScottRochford** [comment]
Hmm... does this belong in Jira?

**Comment by Lakritzator** [comment]
Yeah, it rather belongs in Jira, but I will try to handle it here...
Need to check...

**Comment by ScottRochford** [comment]
Any update?  Should I repost in Jira?

**Comment by wookiefriseur** [comment]
@ScottRochford Hey, in case you are still interested: You can find it [here](https://github.com/greenshot/greenshot/blob/e620cb9/Greenshot/Forms/ImageEditorForm.cs#L834)
